apps/app/messages/en.json

```diff
@@ -161,11 +162,18 @@
     "depositToShort": "Deposit to {name}",
     "withdrawFrom": "Withdraw from {name} on {network}",
     "withdrawFromShort": "Withdraw from {name}",
+    "delegateFrom": "Delegate your {name} on {network}",
+    "delegateFromShort": "Delegate your {name}",
+    "delegateDescription": "The delegated address receives any prizes your {tokens} deposit wins.",
+    "delegatedAddress": "Delegated Address",
+    "changeDelegateAddress": "Change Delegate Address",
+    "updateDelegatedAddress": "Update delegated address",
     "fees": {
       "title": "Estimated Network Fees",
       "approval": "Approval",
       "deposit": "Deposit",
-      "withdrawal": "Withdrawal"
+      "withdrawal": "Withdrawal",
+      "delegate": "Delegate"

```

Comment:
These describe the action that is to be taken, so should be "Delegation" (Like "Withdrawal" instead of a "Withdraw")